src/performance/server.js

```diff
@@ -61,18 +60,12 @@ app.get('/performance/api/v1/environment', (req, res) => {
 */
 app.post('/api/v1/runload', async function (req, res) {
     try {
-        if (loadProcess) {
-            log.error('A load run is already in progress');
-            res.status(409).send("A load run is already in progress");
-            return;
-        }
         if (!req.body.url) {
             log.error('URL is required');
             res.status(500);
```

Comment:
This line is wrong, it should be sendStatus. (The request never terminates if you call it with curl.)